index.html

```diff
@@ -0,0 +1,211 @@
+                        Unfortunately, human-centered design doesn’t always support these principles with the right
+                        intentions.
+                        Instead of seeing a problem in a community, let’s get invited to work with the community.
+                        Instead of
+                        interpreting people’s needs, let’s include them in the process. <strong>Human centered design
+                            implies a
+                            surface
+                            level engagement where the power of design remains with the designer, not the
+                            community.</strong>
+                    </p>
+                </li>
+                <p class="previous-speakers"> <a href="#">View previous speaker events </a></p>
+            </ul>
+
+        </section>
+
+        <section id="register">
+            <h2><span aria-hidden="true">-</span> register <span aria-hidden="true">-</span></h2>
+
+            <form class="registration">
+                <ul class="u-list-reset">
+                    <li>
+                        <label for="name">name:</label>
+                        <input id="name" type="text" name="user_name">
+                    </li>
+                    <li>
+                        <label for="mail">email:</label>
+                        <input id="mail" type="email" name="user_email">
+                    </li>
+                    <li>
+                        <label for="phone-number">phone:</label>
+                        <input type="phone" name="phone" </li> <li>
+                    </li>
+                    <li class="button">
+                        <button class="register-button" class="button-reset" type="submit">Register Now</button>
+                    </li>
+                </ul>
+            </form>
+        </section>
+    </main>
+
+
+    <footer>
+        <nav class="footer-links">
+            <ul class="u-list-reset">
+                <li>
+                    <a href="#details">Details</a>
+                </li>
+                <li>
+                    <a href="#speakers">Speakers</a>
+                </li>
+                <li>
+                    <a href="#register">Register</a>
+                </li>
+                <li>
+                    <a href="#">Previous events</a>
+                </li>
+                <li>
+                    <a href="#">Careers</a>
+                </li>
+                <li>
+                    <a href="#">Contact</a>
+                </li>
+                <li>
+                    <a href="#">Privacy policy</a>
+                </li>
+                <li>
+                    <form class="newsletter">
+                        <label for="mail">Sign up for newsletter:</label>
+                        <input id="mail" type="email" name="user_email">
+                        <button class="newsletter-button" class="button-reset" type="submit">Sign Up!</button>
```

Comment:
Same note about combining your class names:
```suggestion
                        <button class="newsletter-button button-reset" type="submit">Sign Up!</button>
```